### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by adrianne-helpscout**
Ran this README through Grammarly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ The Beacon SDK requires iOS 9.0+.
 
 ### CocoaPods
 
-Beacon is available through [CocoaPods][1]. To install it, simply add the following line to your `Podfile`:
+Beacon is available through [CocoaPods][1]. To install it, add the following line to your `Podfile`:
```
Comment (samsymons) [inline README.md:12]: :+1: I still have a bad habit of adding "simply" to stuff, which I think does more harm than good.

**Review by samsymons — APPROVED** [review]
😍

**Review by ScottPetit — APPROVED** [review]
👍 Looking great.

File: README.md (hunk 1 above)
Comment (adrianne-helpscout) [reply README.md:12]: The grammarly hint is always, sound more confident, remove this word.

**Pull request merged**